crates/oxc_ast_lower/src/lib.rs

```diff
@@ -41,7 +41,9 @@ impl<'a> AstLower<'a> {
         includes: SymbolFlags,
         excludes: SymbolFlags,
     ) -> SymbolId {
-        self.semantic.declare_symbol(span, name, includes, excludes)
+        self.semantic.declare_symbol(
+            span, name, includes, excludes, false, /* this is so much leaked complexity omg */
```

Comment:
This comment is unnecessary, can you comment on `/* what this does */ false` instead?